__tests__/app.test.js

```diff
@@ -30,9 +31,24 @@ describe("GET /api/topics", () => {
 describe("any method: handles all bad paths", () => {
   test("404: responds with bad request for invalid endpoint", () => {
     return request(app)
-    .get("/api/banana")
-    .expect(404).then(( { body}) => {
-      expect(body.msg).toBe("Not Found")
-    })
-  })
-})
\ No newline at end of file
+      .get("/api/banana")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
+      });
+  });
+});
+
+describe("GET /api/", () => {
+  test("200: responds with an object describing all the available endpoints on the API", () => {
+    return request(app)
+      .get("/api/")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toHaveProperty(["GET /api"], expect.any(Object));
+        expect(body).toHaveProperty(["GET /api/topics"], expect.any(Object));
+        expect(body).toHaveProperty(["GET /api/articles"], expect.any(Object));
```

Comment:
You probably don't need these toHaveProperty assertions if you are expecting the whole body to look like endpoints below! I think that expect below is enough 😄

Reply:
Awesome. Should I go back, change them and make another pull request?